statgen/pheget
# Adding short gene names to the region plot title, and toning down the color scheme

## Ticket
n/a

## Purpose
More updates for the UI, mainly to tone down the colors on the buttons and move parts around to make the more important components stand out more (i.e. the main association plots). Also updates the region plot's title (temporary fix until LocusZoom update is published; we will want to update it back to using the title text after https://github.com/statgen/locuszoom/issues/176 is addressed).

## How to test this
In the region view, change anchor gene and tissue, and add new genes and tissues to ensure correct behavior; in the single-variant view, click all the buttons to make sure they still work as expected.

## Deployment / configuration notes
(none)
